src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -26,9 +26,21 @@ export const dashboardReadonlyBadge = {
 };
 
 export const dashboardManagedBadge = {
-  getTooltip: () =>
+  getText: () =>
     i18n.translate('dashboard.badge.managed.tooltip', {
-      defaultMessage: 'Elastic manages this dashboard. Clone it to make changes.',
+      defaultMessage: 'Elastic manages this dashboard.',
+    }),
+  getManagedText: () =>
+    i18n.translate('dashboard.managedContentBadge.text', {
+      defaultMessage: 'Managed',
+    }),
+  getDuplicateText: () =>
+    i18n.translate('dashboard.managedContentPopoverFooterText', {
+      defaultMessage: 'Duplicate this dashboard to make changes',
+    }),
+  getAriaLabel: () =>
+    i18n.translate('dashboard.managedContentBadge.ariaLabel', {
+      defaultMessage: 'Click the badge to duplicate the managed dashboard',
```

Comment:
I'm not sure this `aria-label` is accurate - clicking the badge doesn't duplicate the managed popover, it opens a popover. I think just keeping it as the old tooltip (i.e. "Elastic manages this dashboard. Duplicate it to make changes.") might make the most sense?